### PR TITLE
Python API formatting / code style and extended typing

### PR DESCRIPTION
**Pull request by carlmartus**
- Format with [black](https://pypi.org/project/black/).
- Conform to [pycodestyle](https://pypi.org/project/pycodestyle/).
- Add types to API functions as a method of documentation.

### CODE REVIEW

File: python/remotivelabs-broker/remotivelabs/broker/sync/helper.py [hunk 1]
```diff
@@ -136,15 +155,21 @@ def upload_file(system_stub, path: str, dest_path: str):
...
+    # make sure path is unix style (necessary for windows, and does no harm om
+    # linux)
     upload_iterator = generate_data(
         file, dest_path.replace(ntpath.sep, posixpath.sep), 1000000, sha256
     )
-    response = system_stub.UploadFile(upload_iterator, compression=grpc.Compression.Gzip)
+    response = system_stub.UploadFile(
+        upload_iterator, compression=grpc.Compression.Gzip
+    )
     print("uploaded", path, response)
```
Comment (jrask) [inline python/remotivelabs-broker/remotivelabs/broker/sync/helper.py:166]: Alla print måste bort!

**Comment by jrask** [comment]
All print to stdout must be removed due to compliance with cli

File: python/remotivelabs-broker/remotivelabs/broker/sync/helper.py [hunk 2]
```diff
@@ -86,17 +91,23 @@ def publish_signals(client_id, stub, signals_with_payload, frequency: int = 0):
     except grpc._channel._Rendezvous as err:
         print(err)
 
-def printer(signals):
+
+def printer(signals: Sequence[common_pb2.SignalId]) -> None:
```
Comment (jrask) [inline python/remotivelabs-broker/remotivelabs/broker/sync/helper.py:95]: This is only used if the client program invokes it?

**Review by jrask — CHANGES_REQUESTED** [review]
You can choose to remove stdout now or in a new PR

File: python/remotivelabs-broker/remotivelabs/broker/sync/helper.py (hunk 2 above)
Comment (carlmartus) [reply python/remotivelabs-broker/remotivelabs/broker/sync/helper.py:98]: Yes

**Pull request merged**